src/utils.c

```diff
@@ -144,6 +144,8 @@ static gboolean install_icon_theme_package(const char* package_path)
         argv[1] = "--gzip";
     else if(g_str_has_suffix(package_path, ".tar.bz2"))
         argv[1] = "--bzip2";
+    else if(g_str_has_suffix(package_path, ".tar.xz"))
+        argv[1] = "-J";
```

Comment:
To be consistent I'd use `--xz` instead of its shorthand. You should also not have a merge commit in a pull request. Other than that, the pull request looks good.